src/TrackProgress/SFA.DAS.TrackProgress.Api.AcceptanceTests/ApiTests/CallApi.cs

```diff
@@ -77,681 +55,260 @@ public async Task Track_progress_with_single_matching_apprenticeship_and_no_sand
     [TestCase("rubbish", true)]
     public async Task Track_progress_with_single_matching_apprenticeship_and_different_sandbox_modes(string? isSandbox, bool updateExpected)
     {
-        var mockResponse = GetMockApprenticeshipResponse(1);
-        var singleMockResponse = GetMockSingleApprenticeshipResponse();
-        var courseKsbsResponse = fixture.Create<GetKsbsForCourseOptionResponse>();
-        var validDto = BuildValidProgressDtoContentFromCourseResponse(courseKsbsResponse);
+        var course = A.Course;
+        var apprenticeship = An.Apprenticeship.WithCourse(course);
+        var validDto = BuildValidProgressDtoContentFromCourseResponse(course);
 
-        using (Interceptor.BeginScope())
-        {
-            CommitmentsApi
-                .ForPath("api/apprenticeships")
-                .ForQuery("providerid=12345&searchterm=1&startdate=2020-01-01")
-                .Responds()
-                .WithSystemTextJsonContent(mockResponse)
-                .RegisterWith(Interceptor);
-
-            CommitmentsApi2
-                .ForPath("api/apprenticeships/1")
-                .Responds()
-                .WithSystemTextJsonContent(singleMockResponse)
-                .RegisterWith(Interceptor);
-
-            CoursesApi
-                .ForPath($"/api/courses/standards/{singleMockResponse.StandardUId}/options/{singleMockResponse.Option}/ksbs")
-                .Responds()
-                .WithSystemTextJsonContent(courseKsbsResponse)
-                .RegisterWith(Interceptor);
-
-            TrackProgressInnerApi
-                .ForPost()
-                .ForPath($"apprentice/{12345}/{singleMockResponse.Uln}/2021-01-01/progress")
-                .RegisterWith(Interceptor);
-
-            client.DefaultRequestHeaders.Add(SubscriptionHeaderConstants.ForProviderId, "Provider-12345-TrackProgressOuterApi");
-            client.DefaultRequestHeaders.Add(SubscriptionHeaderConstants.ForSandboxMode, isSandbox);
-
-            var response = await client.PostAsync(
-                $"/apprenticeships/{1}/{"2020-01"}/progress", validDto);
-
-            // TODO Ensure an update is called only when it is expected 
-
-            response.Should().Be201Created();
-        }
+        factory
+            .WithApprenticeship(apprenticeship)
+            .WithCourse(course);
+
+        client.DefaultRequestHeaders.Add(SubscriptionHeaderConstants.ForSandboxMode, isSandbox);
+
+        var response = await client.PostAsync(
+            $"/apprenticeships/{1}/{"2020-01"}/progress", validDto);
+
+        // TODO Ensure an update is called only when it is expected
```

Comment:
We weren't able to test if a call was or wasn't made to the inner API is it worthwhile doing that test now?

Reply:
Removed